src/compiler/performance.ts

```diff
@@ -1,107 +1,110 @@
-/*@internal*/
 namespace ts {

```

Comment:
Do we not want to publicly expose `timestamp`? I guess we don't have a good reason to expose it, so may as well not.